tests/test_vectors/didcomm_messages/tests/common.py

```diff
@@ -1,7 +1,15 @@
-from authlib.common.encoding import json_loads, json_dumps
+from authlib.common.encoding import json_loads, json_dumps, to_unicode, to_bytes, urlsafe_b64decode, urlsafe_b64encode
 
 
 def update(msg, field, new_value):
     msg = json_loads(msg)
     msg[field] = new_value
     return json_dumps(msg)
+
+
+def update_protected(msg, field, new_value):
+    msg = json_loads(msg)
+    protected = json_loads(to_unicode(urlsafe_b64decode(to_bytes(msg["protected"]))))
```

Comment:
[parse_base64url_encoded_json](https://github.com/sicpa-dlab/didcomm-python/blob/a70250476886eb91c66e2170dbaa8df2dd1131a9/didcomm/core/utils.py#L150-L151) does the same.

Please either re-use it or add a TODO (since it's not the only place where we use that long conversion).

Reply:
Added a TODO comment.